src/mcp_server.py

```
@@ -232,198 +328,129 @@ async def _auto_login_if_configured(self):
+                return await self._handle_usermgr_list_group_members(arguments)
+            elif name == "synology_add_user_to_group":
+                return await self._handle_usermgr_add_to_group(arguments)
+            elif name == "synology_remove_user_from_group":
+                return await self._handle_usermgr_remove_from_group(arguments)
+            elif name == "synology_get_user_permissions":
+                return await self._handle_usermgr_get_permissions(arguments)
+            elif name == "synology_set_user_permissions":
+                return await self._handle_usermgr_set_permissions(arguments)
+            else:
+                raise ValueError(f"Unknown tool: {name}")
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)